app/src/main/java/com/android/sample/ui/camera/CameraScanCodeBarScreen.kt

```diff
@@ -147,93 +183,142 @@ fun IngredientOverlay(
+        }
   }
 }
+
+@Composable
+private fun IngredientImage(ingredient: Ingredient) {
+  Box(
+      modifier =
+          Modifier.width(C.Dimension.CameraScanCodeBarScreen.INGREDIENT_DISPLAY_IMAGE_WIDTH.dp)
+              .height(C.Dimension.CameraScanCodeBarScreen.INGREDIENT_DISPLAY_IMAGE_HEIGHT.dp)
+              .border(
+                  width =
+                      C.Dimension.CameraScanCodeBarScreen.INGREDIENT_DISPLAY_IMAGE_BORDER_WIDTH.dp,
+                  color = Color.Black,
+                  shape = RoundedCornerShape(INGREDIENT_DISPLAY_IMAGE_BORDER_RADIUS.dp))
+              .background(MaterialTheme.colorScheme.background),
+  ) {
+    Image(
+        painter =
+            rememberAsyncImagePainter(model = ingredient.images[PRODUCT_FRONT_IMAGE_SMALL_URL]),
+        contentDescription = stringResource(R.string.recipe_image),
+        modifier =
+            Modifier.fillMaxSize()
+                .testTag(RECIPE_IMAGE_1)
+                .clip(RoundedCornerShape(INGREDIENT_DISPLAY_IMAGE_BORDER_RADIUS.dp)),
+        contentScale = ContentScale.Crop,
+    )
+  }
+}
+
+@Composable
+private fun IngredientNameAndBrand(ingredient: Ingredient) {
+  Text(
+      text = ingredient.name,
+      style = MaterialTheme.typography.titleSmall,
+      modifier =
+          Modifier.padding(
+              vertical =
+                  C.Dimension.CameraScanCodeBarScreen.INGREDIENT_DISPLAY_TEXT_NAME_PADDING_V.dp,
+              horizontal =
+                  C.Dimension.CameraScanCodeBarScreen.INGREDIENT_DISPLAY_TEXT_NAME_PADDING_H.dp))
+  Text(
+      text = ingredient.brands ?: "",
+      style = MaterialTheme.typography.bodySmall,
+      modifier =
+          Modifier.padding(
+              vertical =
+                  C.Dimension.CameraScanCodeBarScreen.INGREDIENT_DISPLAY_TEXT_BRAND_PADDING_V.dp,
+              horizontal =
+                  C.Dimension.CameraScanCodeBarScreen.INGREDIENT_DISPLAY_TEXT_BRAND_PADDING_H.dp))
+}
+
+@Composable
+private fun IngredientSelectButton(
+    ingredient: Ingredient,
+    searchIngredientViewModel: SearchIngredientViewModel,
+    navigationActions: NavigationActions
+) {
+  Button(
+      onClick = {
+        searchIngredientViewModel.addIngredient(ingredient)
+        navigationActions.navigateTo(Screen.CREATE_RECIPE_LIST_INGREDIENTS)
+      },
+      modifier =
+          Modifier.padding(
+              vertical =
+                  C.Dimension.CameraScanCodeBarScreen.INGREDIENT_DISPLAY_TEXT_BUTTON_PADDING_V.dp,
+              horizontal =
+                  C.Dimension.CameraScanCodeBarScreen.INGREDIENT_DISPLAY_TEXT_BUTTON_PADDING_H
+                      .dp)) {
+        Text(
+            text = stringResource(R.string.add_to_fridge),
+            style = MaterialTheme.typography.bodySmall,
+            color = MaterialTheme.colorScheme.onPrimary,
+        )
+      }
+}
+
+@Composable
+private fun IngredientNotFoundDisplay() {
+  Log.i("IngredientDisplay", "IngredientNotFoundDisplay")
```

Comment:
Remove this Log or add string into the `C.kt` file pls.

Reply:
Done!